frontend/www/js/omegaup/course/edit.ts

```diff
@@ -0,0 +1,468 @@
+import { omegaup, OmegaUp } from '../omegaup';
+import { types } from '../api_types';
+import * as api from '../api';
+import * as ui from '../ui';
+import T from '../lang';
+import Vue from 'vue';
+import course_Edit from '../components/course/Edit.vue';
+import Sortable from 'sortablejs';
+import Clipboard from 'v-clipboard';
+
+Vue.directive('Sortable', {
+  inserted: (el: HTMLElement, binding) => {
+    new Sortable(el, binding.value || {});
+  },
+});
+Vue.use(Clipboard);
+
+OmegaUp.on('ready', () => {
+  const payload = types.payloadParsers.CourseEditPayload();
+  const courseAlias = payload.course.alias;
+  const courseEdit = new Vue({
+    el: '#main-container',
+    render: function (createElement) {
+      return createElement('omegaup-course-edit', {
+        props: {
+          data: this.data,
+          initialTab: this.initialTab,
+          invalidParameterName: this.invalidParameterName,
+        },
+        on: {
+          'submit-edit-course': (
+            ev: Vue & {
+              alias: string;
+              description: string;
+              finishTime: Date;
+              showScoreboard: boolean;
+              startTime: Date;
+              name: string;
+              school_name: string;
+              school_id: number;
+              basic_information_required: boolean;
+              requests_user_information: boolean;
+              unlimitedDuration: boolean;
+            },
```

Comment:
en vez de `Vue & ...` se puede usar el componente?

```suggestion
            source: course_Edit
```

así si se llegan a cambiar los props / datos de ese componente no hay que también editar este archivo. igual en todos los demás lugares donde hay `Vue & ...`.

Reply:
Listo, mi error era que estaba intentando usar el componente `course_Edit`, pero en realidad el que debe usarse es `course_Form`.